server/controllers/finance/reports/reportAccounts/index.js

```diff
@@ -73,41 +73,41 @@ function getAccountTransactions(accountId, source, dateFrom, dateTo) {
   const params = [accountId];
   let dateCondition = '';
 
-  if(dateFrom && dateTo){
-    dateCondition = `AND DATE(trans_date) BETWEEN ? AND ?`
+  if (dateFrom && dateTo) {
+    dateCondition = 'AND DATE(trans_date) BETWEEN DATE(?) AND DATE(?)';
     params.push(dateFrom, dateTo);
   }
 
-  const csum = `SET @csum := 0`;
+  const csum = 'SET @csum := 0;';
```

Comment:
We should do analysis on if these @ variables are correctly cleaned up and removed. An ideal way to do this might be to keep everything local in a stored procedure/ function.